sparkbox/eslint-config-sparkbox
# chore: Security Updates

This small PR addresses minor security issues with package dependencies.

## Validation
1. Checkout `master`.
1. Run `rm -rf node_modules && npm i`.
1. Run `npm audit`
- [ ] Confirm 8 vulnerabilities are listed.
1. Checkout `chore--security-updates`.
1. Run `rm -rf node_modules && npm I`.
1. Run `npm run test`
1. Run `npm audit`
- [ ] Confirm 3 low severity vulnerabilities are listed.
